src/component/TooltipComponent.tsx

```
@@ -0,0 +1,36 @@
+
+const TooltipComponent = ({
+  content,
+  children,
+  side,
+  delayDuration,
+  sideOffset,
+}: TooltipComponentProps) => {
+  return (
+    <Tooltip delayDuration={delayDuration}>
+      <TooltipTrigger asChild>
+        <button className="typography-body-b bg-primary-0 cursor-pointer rounded-[5px] px-4 py-3 text-white">
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)